week-4/implement/get-ordinal-number.test.js

```diff
@@ -2,3 +2,48 @@
 
 // continue testing and implementing getOrdinalNumber for additional cases
 // Write your tests using Jest - remember to run your tests often for continual feedback
+
+function getOrdinalNumber(number){
+
+    if (number % 10 === 1 && number % 100 !== 11) {
+
+        return number + "st";
+    }
+
+    else if ( number % 10 === 2 && number % 100 !==12){
+
+        return number + 'nd';
+    }
+
+    else if (number % 10 === 3 && number %100 !== 13){
+
+        return number + 'rd'
+    }
+
+    else {
+
+        return number + 'th'
+    }
+}
```

Comment:
A great implementation for the `getOrdinalNumber(number)` function. The only thing I want to mention is, don't use this much of spaces while writing your code, it makes reading code a lot harder for the reader. Or if you prefer using spaces, try to revise your code so that it looks more clean. You can do this either by yourself, or by using a VS Code extension like `Prettier`.